Getting-Started/Setup/Upgrading/general.md

```diff
@@ -83,12 +96,59 @@ The installer will do two things:
 
 We are aware that, currently, the installer is asking you for the database details of a **blank database** while upgrading. In the near future this will be pre-filled with your existing details and the wording will be updated. So no need to be scared. Enter the details of your existing database and Umbraco will upgrade it to the latest version when necessary.
 
+## Run an unattended upgrade
+
+When upgrading your Umbraco project to Umbraco v8.12+ it is possible to enable the upgrade to run unattended. This means that you will not need to run through the installation wizard when upgrading.
+
+Below you will find the steps you need to take in order to upgrade your project unattended.
+
+### Enable the feature
+
+1. Add the `Umbraco.Core.RuntimeState.UpgradeUnattended` key to `appSettings` in your web.config file.
+2. Set the value of the key to `true`.
+
+```xml
+    <add key="Umbraco.Core.RuntimeState.UpgradeUnattended" value="true" />
+</appSettings>
+```
+
+### Check the `ConfigurationStatus`
+
+In order to trigger the actual upgrade, the correct version number needs to be set.
+
+It is important to use the version number of the version that you are upgrading to. If this is not set, the upgrade will not run even if the `UpgradeUnattended` key has been set to `true`.
+
+1. Locate the `ConfigurationStatus` key in the `appSettings` section in your web.config file.
+2. Update the value to match the Umbraco version that you are upgrading to.
+
+```xml
+<add key="Umbraco.Core.ConfigurationStatus" value="x.x.x"/>
+```
+
+### Run the upgrade
+
+With the correct configuration applied, the project will be upgraded on the next boot.
+
+:::note
+While the upgrade processes are running, any requests made to the site will be "put on hold", meaning that no content will be returned before the upgrade is complete.
+:::
+
+#### Boot order
+
+The Runtime level will use `Run` instead of `Upgrade` in order to allow the website to continue to boot up directly after the migration is run, instead of initiating the otherwise required restart.
```

Comment:
@rasmusjp - I've tried adding a note mentioning the requests to the site, and also added one more heading, to sorta split up this section a bit.

I know the phrasing in the new note is a bit strange, but I couldn't really come up with another way of saying it, without using "put on hold" 🤔 Any suggestions are very welcome!!
Let me know what you think!